src/test/integration/adapters-test.js

```diff
@@ -60,4 +60,8 @@ describe('adapters/', function() {
     }
   });
 
+  it('test loading an adapter via plugin mechanism', async () => {
+
```

Comment:
Is this meant to be blank?

Reply:
I was originally going to put the tests here and decided to move it. I was supposed to checkout the previous version of this file, and forgot to.